eyelid-match-ops/src/primitives/yashe/conf.rs

```diff
@@ -65,8 +101,11 @@ impl YasheConf for FullRes {
 #[cfg(tiny_poly)]
 impl YasheConf for TinyTest {
     /// Limited to the modulus of the underlying `Coeff` type.
-    const T: u64 = 7;
+    const T: u64 = 4;
 
     /// Limited to 1/6 of the modulus, so that the sampled values are valid within 6 sigmas.
-    const DELTA: f64 = 0.9;
+    const KEY_DELTA: f64 = 0.6;
+
+    /// Limited to 1/3 of KEY_DELTA, so that the error is small enough for valid decryption.
+    const ERROR_DELTA: f64 = 0.2;
```

Comment:
```suggestion
    const KEY_DELTA: f64 = 0.5;

    /// Limited to 1/3 of KEY_DELTA, so that the error is small enough for valid decryption.
    const ERROR_DELTA: f64 = 0.16;
```